### DISCUSSION TITLE
After login with Openlogin how can I login on torus wallet same time on same website

### DISCUSSION THREAD

**Discussion by Dibs0845** [Web3Auth/Web3Auth#792] [Help]
After login with Openlogin how can I login on torus wallet same time on same website

**Comment by shahbaz17** [comment]
Hey @Dibs0845 

Does this mean using the torus wallet plugin inside a dapp? So upon login with socials, it gets logged in on the torus wallet plugin?

Or Can you please try explaining the use case here?
